src/Gelf4net/packages.config

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="log4net" version="2.0.3" targetFramework="net451" />
-  <package id="Newtonsoft.Json" version="4.5.10" targetFramework="net40" />
-  <package id="Newtonsoft.Json" version="4.5.11" targetFramework="net40" />
-  <package id="RabbitMQ.Client" version="3.0.0" targetFramework="net40" />
-  <package id="RabbitMQ.ServiceModel" version="3.0.0" targetFramework="net40" />
+  <package id="log4net" version="2.0.12" targetFramework="net472" />
+  <package id="Newtonsoft.Json" version="13.0.1" targetFramework="net472" />

```

Comment:
That is a very large Newtonsoft jump.  Are we sure this is good?

Also, we said we are NOT updating dependents for this because they have already been upgraded transitive dependency wise explicitly in the applications?

Reply:
We have consumers of this library already on Newtonsoft 13.0.1. Yes no need to deploy as consumers have all already updated the problematic log4net reference.